assets/js/modules/reader-revenue-manager/components/settings/SettingsView.js

```diff
@@ -46,7 +82,7 @@ export default function SettingsView() {
 					</p>
 				</div>
 			</div>
-			<PublicationOnboardingStateNotice />
+			{ hasModuleAccess && <PublicationOnboardingStateNotice /> }
```

Comment:
To be consistent with the settings edit screen, we're displaying the onboarding state notice in the settings view screen only if the user has module access. I believe we wouldn't want someone without module access to be able to click on the Publisher Center CTAs.